apps/platform/trpc/routers/userRouter/securityRouter.ts

```diff
@@ -1334,6 +1337,17 @@ export const securityRouter = router({
           .where(inArray(convoEntrySeenTimestamps.orgId, orgIdsArray))
       ]);
 
+      const orgPublicIdsArray = orgsQuery.map((org) => org.publicId);
```

Comment:
we should check if the org has a postal config row in the postal configs table, beofore sending the request to mailbridge